server.js

```diff
@@ -23,7 +23,7 @@ http.createServer( function( request, response ) {
 		return response.end()
 	}
 
-	tachyon( config.region, config.bucket, decodeURI( params.pathname.substr(1) ), params.query, function( err, data, info ) {
+	tachyon.s3( config.region, config.bucket, decodeURI( params.pathname.substr(1) ), params.query, function( err, data, info ) {
```

Comment:
I believe that's already done in the `lambda-handler.js` - makes a call to `tachyon.s3()`